Sources/Commands/SwiftRunTool.swift

```diff
@@ -122,7 +122,7 @@ public struct SwiftRunTool: SwiftCommand {
                 buildParameters: buildParameters,
                 cacheBuildManifest: false,
                 packageGraphLoader: graphLoader,
-                pluginInvoker: { _ in [:] },
+                buildToolPluginInvoker: { _ in [:] },
```

Comment:
would this not be a problem for running your app in REPL? not urgent to fix now, but if it is a problem we should add a FIXME and radar

Reply:
Yes, this was a TODO left over from earlier implementation.  Should add a Radar number for sure.